app/src/main/res/layout/home_activity.xml

```diff
@@ -18,7 +18,7 @@
     <FrameLayout
       android:id="@+id/home_fragment_placeholder"
       android:layout_width="match_parent"
-      android:layout_height="0dp"
+      android:layout_height="@dimen/home_activity_frame_layout_height"
```

Comment:
You can directly hard code this here as `0dp` and remove the dimens value that you have created from `dimens.xml` file.

Reply:
@rt4914 I am done with the requested changes... PTAL